supabase/functions/_shared/helpers.ts

```diff
@@ -1,20 +1,51 @@
-import { corsHeaders } from "./cors.ts";
+import HandlebarsJS from 'https://esm.sh/handlebars';
+import Mustache from 'https://esm.sh/mustache';
+import { corsHeaders } from './cors.ts';
 
 export const returnPostgresError = (error: any) => {
-  return new Response(JSON.stringify({ error }), {
-    headers: {
-      ...corsHeaders,
-      "Content-Type": "application/json",
-    },
-    status: 400,
-  });
+    return new Response(JSON.stringify({ error }), {
+        headers: {
+            ...corsHeaders,
+            'Content-Type': 'application/json',
+        },
+        status: 400,
+    });
 };
 
 export const handlebarsHelpers = {
-  urlencode: function(s: string) {
-    return encodeURIComponent(s)
-  },
-  basicauth: function(user: string, password: string) {
-    return btoa(`${user}:${password}`);
-  }
-}
+    urlencode: function (s: string) {
+        return encodeURIComponent(s);
+    },
+    basicauth: function (user: string, password: string) {
+        return btoa(`${user}:${password}`);
+    },
+};
+
+export const compileTemplate = (template: string, data: any, connector_id: string) => {
+    (HandlebarsJS as any).registerHelper(handlebarsHelpers);
+
+    const compiledTemplate = (HandlebarsJS as any).compile(template);
+    const handlebarsOutput = compiledTemplate(data);
+
+    try {
+        // Override escaping globally for mustache
+        (Mustache.escape as any) = function (text: any) {
+            return text;
+        };
```

Comment:
I think we can just let Mustache handle escaping as it usually does and if it's an error, then we resolve that as part of our transition, what do you think?

Reply:
That works for me... cause this does not escape everything exactly the same anyway.

Reply:
Removed.